src/test/java/Step_Definition/purchase5.java

```diff
@@ -52,12 +57,14 @@ public void iPlaceAnOrder() throws InterruptedException {
         page.selectpayviacheck();
         page.clickconfirmorder();
         page.clickbackOrder();
+
     }
 
     @Then("^I should see my order history and reference$")
     public void iShouldSeeMyOrderHistoryAndReference() {
         String ExpectedMessage = "ORDER HISTORY";
```

Comment:
Please assert your successful order with the success message. This order history doesnt really confirm the order that the last test run made...